### DISCUSSION TITLE
classes/class_audioeffecthighshelffilter

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#91] [User-contributed notes]
# classes/class_audioeffecthighshelffilter

Inherits: AudioEffectFilter< AudioEffect< Resource< RefCounted< Object Adds a high-shelf filter to the audio bus. Description: Reduces all frequencies above the AudioEffectFilter.cutoff_hz. Tutoria...

https://docs.godotengine.org/en/stable/classes/class_audioeffecthighshelffilter.html

**Comment by mariowix** [comment]
This effect does not work on HTML5 exports with 4.3.stable
It used to work with 4.2

**Reply by Calinou** [reply]
Audio effects are not supported when using [sample playback](https://github.com/godotengine/godot/pull/91382), which is the default on the web platform since 4.3. This allows for low latency playback even when exporting without threads.

You can change the playback type in the AudioStreamPlayer properties, or in the project settings to set a default. Keep in mind using Godot's own playback will have high latency when exporting to the web *without* threads. To get low latency audio while being able to use Godot's audio effects on the web, you must export with threads enabled (see [serving considerations](https://docs.godotengine.org/en/latest/tutorials/export/exporting_for_web.html#serving-the-files)).

If you only want the audio effect to apply to the music, you could make its AudioStreamPlayer use Godot's own playback system even on the web (since latency for music playback is usually not a concern).
